evennia/utils/opclasses.py

```diff
@@ -0,0 +1,319 @@
+import datetime as _dt
+from evennia.utils.ansi import ANSIString as _ANSI
+from evennia.utils.validfuncs import _TZ_DICT
+from evennia.utils.valid import VALID_HANDLER as _VAL
+
+
+class _BaseOption(object):
+    """
+    Abstract Class to deal with encapsulating individual Options. An Option has a name/key, a description
+    to display in relevant commands and menus, and a default value. It saves to the owner's Attributes using
+    its Handler's save category.
+
+    Designed to be extremely overloadable as some options can be cantankerous.
+
+    Properties:
+        expect_type (str): What users will see this as asking for. Example: Color or email.
```

Comment:
This seems superfluous and also adds extra boiler plate for every option type. We are already setting both a name, a description *and* a named validator-function for this option. Surely one of those resources should allow us to deduct and display what type we are expecting?